StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/NetworkingOTPView/NetworkingOTPView.swift

```diff
@@ -129,7 +129,6 @@ final class NetworkingOTPView: UIView {
 
     private func userDidEnterValidOTPCode(_ otpCode: String) {
         otpTextField.resignFirstResponder()
-        // TODO(kgaidis): consider implementing a loading/grayed-out state for `otpTextField`
```

Comment:
@kgaidis-stripe  - Removed this as it's implemented, lmk if I should keep it :)

Reply:
if you will make changes, feel free to keep this, but its not a big deal

the core thing I didn't understand is that this was mostly an Identity change, so Financial Connections actually is unaffected at this current moment without more code changes